test/performance-split-words-time.ts

```diff
@@ -0,0 +1,39 @@
+import test, { ExecutionContext } from "ava";
```

Comment:
Overall, I would be happier if this file was NOT an Ava test file 😂 

It does not look or act like an _Ava_ test — it is okay to test performance, but I'm not sure what hooking it into the Ava test framework is really benefiting anything. This file hardly uses Ava's features, and there are no "real" assertions. It also seems to not provide any feedback that can be automatically assessed by tools (e.g., for detecting regressions).

As such, I think it is inappropriate for this file to patch into Ava. It can be its own standalone file, just fine.